### PR TITLE
Add testing.md files to each microservice

### PR DESCRIPTION
**Pull request by pragyamehta**
This will hold information needed to debug locally

### CODE REVIEW

File: samples/BikeSharingApp/ReservationEngine/TESTING.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### Local Debugging pre-requisites
+* ```dotnet restore```
+* [Debugging dotnet core app in VS Code](https://docs.microsoft.com/en-us/dotnet/core/tutorials/debugging-with-visual-studio-code)
+
+<br/>
+
+### Additional information
+* Service port : 49308
```
Comment (daniv-msft) [inline samples/BikeSharingApp/ReservationEngine/TESTING.md:8]: >49308 [](start = 17, length = 5)

I believe this one should be "5000", as we just take the default .NET Core port for the ReservationEngine. I think this port is used by IIS, but it's not the one we listen to locally by default.

File: samples/BikeSharingApp/Bikes/TESTING.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+### Local Debugging pre-requisites
+* ```npm install```
+* [Debugging node js app in VS Code](https://code.visualstudio.com/docs/nodejs/nodejs-debugging)
```
Comment (daniv-msft) [inline samples/BikeSharingApp/Bikes/TESTING.md:3]: >node js [](start = 13, length = 7)

Nit: "Node.js" to respect the format they use for their name?

Comment (pragyamehta) [reply samples/BikeSharingApp/Bikes/TESTING.md]: Updated

File: samples/BikeSharingApp/ReservationEngine/TESTING.md (hunk 1 above)
Comment (pragyamehta) [reply samples/BikeSharingApp/ReservationEngine/TESTING.md]: I think you're right... updated

File: samples/BikeSharingApp/Gateway/TESTING.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+### Local Debugging pre-requisites
+* ```dotnet restore```
+* [Debugging dotnet core app in VS Code](https://docs.microsoft.com/en-us/dotnet/core/tutorials/debugging-with-visual-studio-code)
```
Comment (daniv-msft) [inline samples/BikeSharingApp/Gateway/TESTING.md:3]: >dotnet core [](start = 13, length = 11)

Nit: similar comment, should this be ".NET Core" to match the official name?

**Pull request merged**